sdcm/nemesis_registry.py

```diff
@@ -90,7 +93,9 @@ def gather_properties(self) -> Tuple[Dict[SourceType, Dict[str, bool]], Dict[str
         method_properties = {}
         for subclass in self.get_subclasses():
             properties_list = {}
-            for attribute in subclass.__dict__.keys():
+            keys = dict(**self.flag_class.__dict__)
```

Comment:
Please, rename the `keys` dictionary to more correct name.
For example - `nemesis_params`, `nemesis_flags` or `nemesis_filters`.